distribution-spec/live/badges.md

```diff
@@ -0,0 +1,6 @@
+| Type | Status |
+|------|--------|
+| OCI - Push | [![OCI Push](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-push.yml/badge.svg)](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-push.yml)|
+| OCI - Pull | [![OCI Pull](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-pull.yml/badge.svg)](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-pull.yml)|
+| OCI - Content Management | [![OCI Content Management](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-content-management.yml/badge.svg)](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-content-management.yml)|
+| OCI - Content Discovery | [![OCI Content Discovery](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-content-discovery.yml/badge.svg)](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-content-discovery.yml)|
```

Comment:
Please put one badge per line (no additional markdown)

For example please see https://github.com/opencontainers/oci-conformance/blob/main/distribution-spec/live/zot/badges.md

Additionally, this file should be nested under the `$dir`. For example, `distribution-spec/live/openregistry/badges.md`

Reply:
Moved the file inside openregistry/ and removed extra markdown